src/data/map.js

```diff
@@ -1265,6 +1266,7 @@ const getSearchData = async (lat, lon, id, value, iconStyle) => {
             FROM gym
             WHERE LOWER(name) LIKE '%${sanitizedValue}%'
             `;
+            useManualDb = dbOptions.includes('gym') ? false : true;
```

Comment:
Simplify check (remove `!` if we import manualdb useFor)
```suggestion
            useManualDb = !dbOptions.includes('gym');
```